Doc/library/concurrent.futures.rst

```diff
@@ -54,11 +54,16 @@ Executor Objects
        specified by setting *chunksize* to a positive integer. For very long
        iterables, using a large value for *chunksize* can significantly improve
        performance compared to the default size of 1. With :class:`ThreadPoolExecutor`,
-       *chunksize* has no effect.
+       *chunksize* has no effect. By default, a reasonable number of tasks are
+       queued beyond the number of workers, an explicit *prefetch* count may be
+       provided to specify how many extra tasks should be queued.
 
```

Comment:
Yea, queueing fewer tasks than the number of workers isn't as useful as I thought because the executor isn't really intended for load balancing.